examples/twittermap/web/app/views/twittermap/index.scala.html

```diff
@@ -32,6 +32,10 @@
     </button>
   </div>
 
+  <div id="vis">
```

Comment:
Maye change a more meaningful id than `vis`, seems you didn't use it in CSS file, maybe we can delete this id.